Using std::size_t to allow compiler to find size_t.
Will not send upstream.

--- IlmImf/ImfZip.h	2016-11-17 14:32:08.409507123 +0000
+++ IlmImf/ImfZip.h	2016-11-17 14:32:23.052753997 +0000
@@ -39,6 +39,7 @@
 #include "ImfNamespace.h"
 
 #include <cstddef>
+using std::size_t;
 
 OPENEXR_IMF_INTERNAL_NAMESPACE_HEADER_ENTER
 
